Database/Corrections/sodQuestFixes.lua

```diff
@@ -229,6 +229,24 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -425170,
             [questKeys.zoneOrSort] = sortKeys.MAGE,
         },
+        [78132] = {
+            [questKeys.startedBy] = {nil,{409758}},
```

Comment:
```suggestion
            [questKeys.objectives] = {nil,nil,{{209872}}},
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5416/commits/b8b66e1032045f73d133207f1a9eb4ae59612d21